modules/Date.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Instead of disabling `eslint`, it will be nice to fix the issue. Checkout this [article](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export).